tsconfig.json

```diff
@@ -15,8 +15,7 @@
     "allowJs": false,
     "declaration": true,
     "noEmitOnError": true,
-    "allowSyntheticDefaultImports": true,
-    "baseUrl": "./src"
```

Comment:
This should be enough to prevent people from using the previous syntax, correct ?

Reply:
Yes, without `baseUrl` TS compiler will used node's module resolution.